server.go

```diff
@@ -1015,7 +1019,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	}
 
 	s.authGossiper = discovery.New(discovery.Config{
-		Router:                s.chanRouter,
+		Router:                s.graphBuilder,
```

Comment:
I think we can rename this field while we're at it. Could just be `ChanGraph` or w/e.